ecosystem/nft-metadata-crawler/nft-metadata-crawler-parser/migrations/2023-06-27-001532_create_tables/up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS nft_metadata_crawler_uris (
+  token_uri VARCHAR UNIQUE PRIMARY KEY NOT NULL,
+  raw_image_uri VARCHAR,
+  raw_animation_uri VARCHAR,
+  cdn_json_uri VARCHAR,
+  cdn_image_uri VARCHAR,
+  cdn_animation_uri VARCHAR,
+  json_parser_retry_count INT NOT NULL,
+  image_optimizer_retry_count INT NOT NULL,
+  animation_optimizer_retry_count INT NOT NULL,
+  last_updated TIMESTAMP NOT NULL DEFAULT NOW()
```

Comment:
now that I think about it, can we make this inserted_at and add an index on this similar to other tables in indexer? We might want to ingest into bigquery so it'd be nice to follow the same pattern.